src/BreakfastSerializer/Property/DepthTraversableProperty.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
```

Comment:
good point I better add in some more to be sure :D

On Fri, Jul 17, 2015 at 8:49 PM, Eddie Abou-Jaoude <notifications@github.com
> wrote:

> In src/BreakfastSerializer/Property/DepthTraversableProperty.php
> <https://github.com/brettminnie/breakfast-serializer/pull/20#discussion_r34924781>
> :
>
> > @@ -0,0 +1,99 @@
> > +<?php
> > +
>
> Enough spaces?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brettminnie/breakfast-serializer/pull/20/files#r34924781>
> .
>